src/components/TimelineEvent.js

```diff
@@ -2,9 +2,14 @@ import React from 'react';
 import './TimelineEvent.css';
 import Timestamp from './Timestamp';
 
-const TimelineEvent = () => {
-  // Fill in your code here
-  return;
+const TimelineEvent = (props) => {
+  return (
+    <section class="timeline-event">
+      <h4 class="event-person">{props.person}</h4>
+      <p class="event-status">{props.status}</p>
+      <p class="event-time"><Timestamp time={props.timestamp} /></p>
```

Comment:
It appears that this is not actually working. The date stamps would be something like a year ago (I ran into this problem too). Consider taking a look at the JSON file. The key for timeStamp has a capital 's'. I would play around with this to see if you can see the timestamp based off of the dates in the JSON file. :)